core/src/main/java/io/questdb/cairo/wal/WalWriter.java

```diff
@@ -413,9 +413,14 @@ public boolean inTransaction() {
         return segmentRowCount > currentTxnStartRowNum;
     }
 
-    public void invalidate(boolean invalid, @Nullable CharSequence invalidationReason) {
+    public void invalidate(
```

Comment:
The method's name is misleading. What does it invalidate? WAL table as the name suggests?

`invalidateMatView` would be much more obvious.

Reply:
renamed: https://github.com/questdb/questdb/pull/5519/commits/3b0722404833a5b508a78aebc477d80ddd3ccc5e